lhctl/cmd/user_task_run.go

```diff
@@ -151,6 +151,83 @@ var getUserTaskRunCmd = &cobra.Command{
 	},
 }
 
+var saveUserTaskRunProgressCmd = &cobra.Command{
+	Use:   "saveUserTaskRun",
```

Comment:
To make this fit with the rest of `lhctl`'s design patterns, the Verb and the Object Noun should be separate. This means it should be:

```
lhctl save userTaskRun <wfRunId> <userTaskGuid>
```

To do that, you'll need to add a `save.go` and a `saveCmd`.

Reply:
I recognized this design pattern when developing this but chose not to follow it because I think is bad UX having one single command inside of a verb command.

It adds too many layers of `--help` menus that are unnecessary. 2 layers vs 3.

Long term, it will make adding more `save` commands slightly easier but in the short term it will cost developers more time to discover and use this command.